### PR TITLE
feat(frontend): signer "alert" symbol

### PR DESCRIPTION
**Pull request by peterpeterparker**
# Motivation

When a call canister succeed or fails, we want to display the result with a pictogram to the user.

# Changes

- Add a `SignerAlert` component that is either a checkmark or a close symbol.

# Screenshots

Capture d’écran 2024-10-16 à 18 52 25 copie
Capture d’écran 2024-10-16 à 18 52 42 copie

### CODE REVIEW

File: src/frontend/src/lib/components/signer/SignerAlert.svelte [hunk 1]
```diff
@@ -0,0 +1,20 @@
+<script lang="ts">
+	import type { ComponentType } from 'svelte';
+	import IconCheck from '$lib/components/icons/IconCheck.svelte';
+	import IconClose from '$lib/components/icons/IconClose.svelte';
+
+	export let type: 'ok' | 'error';
```
Comment (AntonioVentilii-DFINITY) [inline src/frontend/src/lib/components/signer/SignerAlert.svelte:6]: not a call to action: maybe something less built-in as `type`? `alertType`, or similar

**Review by AntonioVentilii-DFINITY — APPROVED** [review]
LGTM!

File: src/frontend/src/lib/components/signer/SignerAlert.svelte (hunk 1 above)
Comment (peterpeterparker) [reply src/frontend/src/lib/components/signer/SignerAlert.svelte:6]: Good call. `alertType` sounds good.

Comment (peterpeterparker) [reply src/frontend/src/lib/components/signer/SignerAlert.svelte:6]: Done.

**Pull request merged**